deploy/load-test/stack/app.yml

```diff
@@ -0,0 +1,204 @@
+version: '3.4'
```

Comment:
This file should be split for each service, and each service must be placed in its proper context directory, just as with storages.